templates/index.html

```diff
@@ -36,21 +17,22 @@
 
     <div class="meetup">
 
-      <h3 class='meetup-name'><a href="{{ meetup.meetup_link }}">{{ meetup.name }}</a></h3>
+      <h3 class='meetup-name'><a href="{{ meetup.get_meetup_link }}">{{ meetup.get_meetup_name }}</a></h3>
 
-      {% if meetup.next_event_topic %}
+      {% if meetup.get_meetup_next_event_topic %}
         <div class='meetup-description'>
           <strong class='subheader'>Next Event:</strong>
-          <p>{{ meetup.next_event_topic }} - {{ meetup.datetime}} at
-          <a href="{{ meetup.map_link }}">{{ meetup.venue }}</a></p>
+          <p>{{ meetup.get_meetup_next_event_topic }} - {{ meetup.get_meetup_datetime}} at
```

Comment:
Is this actually a paragraph, or are you just using a `<p>` tag to create a block-level element? Is there another semantic element that might be more appropriate? https://developer.mozilla.org/en-US/docs/Web/HTML/Element